cmd/tapcli/proofs.go

```diff
@@ -81,6 +86,68 @@ func verifyProof(ctx *cli.Context) error {
 	return nil
 }
 
+var decodeProofCommand = cli.Command{
+	Name:        "decode",
+	ShortName:   "d",
+	Description: "decode a Taproot Asset proof",
```

Comment:
This should be the `Usage` field instead. See recent changes here: https://github.com/lightninglabs/taproot-assets/pull/359/commits/1607bf06d0801c3ebc72d383a7e34980376c4029